internal/controller/stages/regular_stages.go

```diff
@@ -322,6 +322,11 @@ func (r *RegularStageReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return ctrl.Result{}, nil
 	}
 
+	if stage.GetLabels()[kargoapi.LabelKeyShard] != r.cfg.ShardName {
+		logger.Debug("ignoring Stage because it is not in scope")
```

Comment:
I think this wording is more clear in meaning that scope. Applies in the Warehouse reconciler as well.

```suggestion
		logger.Debug("ignoring Stage because it is is not assigned to this shard")
```